assets/js/pages/Users/UserForm.jsx

```diff
@@ -100,11 +103,14 @@ function UserForm({
     const user = {
       fullname: fullNameState,
       email: emailAddressState,
-      username: usernameState,
-      password: passwordState,
-      password_confirmation: confirmPasswordState,
       enabled: statusState === USER_ENABLED,
+      ...(!editing && { username: usernameState }),
```

Comment:
This is a cool usage of the spread operator xD

Reply:
basically, in patches, we don't want to send all the fields, so this is a nice way of conditionally adding them